Makefile

```diff
@@ -91,11 +91,18 @@ GO_PROJECT := sigs.k8s.io/$(PROJECT)
 LDVARS := \
 	-X $(GO_PROJECT)/internal/pkg/version.buildDate=$(BUILD_DATE) \
 	-X $(GO_PROJECT)/internal/pkg/version.version=$(VERSION)
+STATIC_LINK ?= yes
```

Comment:
Why not default this to true when building the UBI container?

Reply:
The default is true (same as it is now), did you mean false?

I /thought/ that UBI didn't have the required devel packages, but it seems they do now. I'll try it, thanks!

Reply:
OK, turns out that it was the static packages that were missing, not devel packages. I was confused!